packages/turf-union/test.js

```diff
@@ -6,8 +6,6 @@ const write = require('write-json-file');
 const combine = require('@turf/combine').default;
 const union = require('./dist/js/index.js').default;
 
-const SKIP=["union4.geojson"];
```

Comment:
Can you leave this skipping deleted, but keep the union4.geojson tests? The test should be fine but it was skipped here because it was failing because of the `martinez-polygon-clipping` issues.